lib/pages/faq/QuestionsList.dart

```diff
@@ -0,0 +1,123 @@
+    });
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    List<Question> filteredList = widget.questions
+        .where((question) =>
+            widget.filter.split(" ").where((element) => element != "").fold(
+                true,
+                (previousValue, filter) =>
+                    previousValue &&
+                    question.question.toLowerCase().contains(filter)) &&
+            (activeCategory == "" ? true : question.category == activeCategory))
+        .toList();
+
+    List categoryList = widget.categories
+        .map((category) => Padding(
+              padding: const EdgeInsets.symmetric(horizontal: 3.0),
+              child: Container(
+                width: 120,
+                child: Selectable(
+                  controller: _controllers[category],
+                  label: category,
+                  initiallySelected: false,
+                  onSelected: (selection) {
+                    setState(() {
+                      activeCategory = "";
+                      if (selection) {
+                        _controllers.values.forEach((element) {
+                          element.deselect();
+                        });
+                        _controllers[category].select();
+                        activeCategory = category;
+                      }
+                    });
+                  },
+                ),
+              ),
+            ))
+        .toList();
+
+    List<String> filteredWords =
+        widget.filter.split(" ").where((element) => element != "").toList();
+
+    return Column(
+      children: [
+        Container(
+          height: 40,
+          child: ListView(
+            scrollDirection: Axis.horizontal,
+            children: categoryList,
+          ),
+        ),
+        Padding(
+          padding: EdgeInsets.only(top: 12.0),
+          child: ListView.builder(
+            shrinkWrap: true,
+            physics: NeverScrollableScrollPhysics(),
+            itemCount: filteredList.length,
+            itemBuilder: (context, index) {
+              return ExpansionTile(
+                title: filteredWords.isNotEmpty
+                    ? DynamicTextHighlighting(
+                        text: filteredList[index].question,
+                        style: Theme.of(context).textTheme.subtitle1,
+                        highlights: filteredWords,
+                        color: Colors.blue,
+                        caseSensitive: false,
+                      )
+                    : Text(
+                        filteredList[index].question,
+                        style: Theme.of(context).textTheme.subtitle1,
+                      ),
+                children: [
+                  Padding(
+                    padding: const EdgeInsets.all(8.0),
+                    child: MarkdownBody(
+                      data: filteredList[index].answer.replaceAll('\\n', '\n'),
```

Comment:
What is the point of the `replaceAll` call?

Reply:
Firebase database doesn't recognize endline and i added \n for them. Without `replaceAll`  for some reason dart will prin `/n` and not print a new line.

Reply:
The console apparently strips newlines when displaying strings, but they should be read correctly in the app without you needing to write them as \n. See this issue: https://github.com/firebase/firebase-js-sdk/issues/2366

You should be able to change the answers in the database to contain actual newlines (even if they don't show in the console), and remove the `replaceAll` call here.

Reply:
i tried and the problem was that they were not shown in the app

Reply:
That's odd. Add a comment in the code explaining why this is necessary and maybe we'll find a cleaner fix later.